features/guidance/roundabout.feature

```diff
@@ -707,3 +744,27 @@ Feature: Basic Roundabout
         When I route I should get
             | waypoints | route          | turns                                                                           |
             | a,h       | ab,ef,ef,fh,fh | depart,roundabout-exit-4,notification slight right,notification straight,arrive |
+
+
+     Scenario: Drive through roundabout
+        Given the node map
+           """
+              a
+            b e d  f
+              c
+            g   h
+           """
+
+        And the ways
+           | nodes | junction   | oneway |
+           | abcda | roundabout | yes    |
+           | edf   |            |        |
+           | gch   |            | yes    |
+
+        When I route I should get
+           | waypoints | bearings | route       | turns                           |
+           | e,f       | 90 90    | edf,edf,edf | depart,roundabout-exit-1,arrive |
```

Comment:
This is not necessarily related to the PR at hand, but I am unsure whether we wan't to count this as an exit. Do you see a possibility that this could be changed to a `TurnType::Suppressed`? It would require an adjustment to the roundabout handler.
Probably a topic for a different PR, though.